### DISCUSSION TITLE
Equillibrium check

### DISCUSSION THREAD

**Discussion by valeriabriones** [ua-snap/dvm-dos-tem#676] [General]
Curious if anyone has recommendations for how to successfully run the equillibrium_check.py script in the ac-refac branch? So far I've imported drivers.MadsTEMDriver in place of TEM on line 21: "import drivers.MadsTEMDriver as TEM"
 and have then changed on line 24: "dvmdostem=TEM.MadsTEMDriver()" along with changing directory paths and variables.
I keep getting an error with seedpath and config keys, which I've tried to troubleshoot but haven't been successful. Have others had luck?

**Comment by tobeycarman** [comment]
@valeriabriones no one has used it that I know of. There is an open issue #666 to address it. I have some ideas of how to implement that I will try to record over there.
